client/extensions/woocommerce/app/products/product-form-simple-card.js

```diff
@@ -0,0 +1,155 @@
+		editProduct( product, { backorders: e.target.value } );
+	};
+
+	// TODO Pull in currency and currency position.
+	const renderPrice = () => (
+		<Card className="products__product-form-price">
+			<FormLabel>{ translate( 'Price' ) }</FormLabel>
+			<FormCurrencyInput
+				currencySymbolPrefix="$"
+				name="price"
+				value={ product.regular_price || '' }
+				onChange={ setPrice }
+				size="4"
+			/>
+		</Card>
+	);
+
+	// TODO pull in dimension and weight units.
+	const renderDeliveryDetails = () => (
+		<Card className="products__product-form-delivery-details">
+			<div className="products__product-dimensions-weight">
+				<FormFieldSet className="products__product-dimensions-input">
+					<FormLabel>{ translate( 'Dimensions' ) }</FormLabel>
+					<FormDimensionsInput
+						unit={ translate( 'in' ) }
+						dimensions={ product.dimensions }
+						onChange={ setDimension }
+					/>
+				</FormFieldSet>
+				<FormFieldSet className="products__product-weight-input">
+					<FormLabel>{ translate( 'Weight' ) }</FormLabel>
+					<FormTextInputWithAffixes
+						name="weight"
+						type="number"
+						min="0"
+						suffix="g"
+						value={ product.weight || '' }
+						onChange={ setWeight }
+						size="4"
+					/>
+				</FormFieldSet>
+			</div>
+			<FormSettingExplanation>{ translate(
+				'Shipping services will use this data to provide accurate rates.'
+			) }</FormSettingExplanation>
+		</Card>
+	);
+
+	const renderStock = () => (
+		<Card className="products__product-form-stock">
+			<div className="products__product-manage-stock-wrapper">
+				<FormLabel>{ translate( 'Manage stock' ) }</FormLabel>
+				<div className="products__product-manage-stock">
+					<div className="products__product-manage-stock-checkbox">
+						<FormCheckbox
+							checked={ Boolean( product.manage_stock ) }
+							name="manage_stock"
+							onChange={ toggleStock } />
+					</div>
+
+					{ product.manage_stock && (
+						<FormTextInput
+							name="stock_quantity"
+							value={ product.stock_quantity || '' }
+							type="number"
+							min="0"
+							onChange={ setStockQuantity }
+							placeholder={ translate( 'Quantity' ) } />
+					) }
+				</div>
+			</div>
+			{ product.manage_stock && (
+				<div className="products__product-backorders-wrapper">
+					<FormLabel>{ translate( 'Backorders' ) }</FormLabel>
+					<FormSelect name="backorders" onChange={ setBackorders } value={ product.backorders || 'no' } >
+						<option value="no">{ translate( 'Do not allow' ) }</option>
+						<option value="notify">{ translate( 'Allow, but notify customer' ) }</option>
+						<option value="yes">{ translate( 'Allow' ) }</option>
+					</FormSelect>
+
+					<FormSettingExplanation>{ translate(
+						'Backorders allow customers to purchase products that are out of stock.'
+					) }</FormSettingExplanation>
+				</div>
+			) }
+		</Card>
+	);
+
+	return (
+		<div>
+			{ renderPrice() }
+			{ renderDeliveryDetails() }
+			{ renderStock() }
+		</div>
+	);
```

Comment:
I like how simple the final render looks here.
Just keep in mind if these things are broken out, you'll have to pass them parameters then.